sim.py

```diff
@@ -162,15 +162,15 @@ def buildSim(self, ldr):
         self.map.buildMapGrid()
 
         # Create the map border
-        edge_obj_id = self.map.getData("edge_obj_id")
+        edge_obj_id = self.map.getData("edge_obj_id")  # Point B
```

Comment:
What is the significance of these "Point" comments? Are they meant to be left in the final product?

Reply:
nope my bad